docs/bbs/pam.md

```diff
@@ -30,3 +30,73 @@ Animations in Birth by Sleep do not need to be aligned in any way.
 | 06 | byte | Bone count, must match bone count of target model.|
 | 08 | uint16_t | Animation frame count.|
 | 0A | uint16_t | Loop to frame, determines the point the animation loops to.|
+
+## Animation Data
+The animation data consists of separate channels for each animated property. There can be maximum 9 channels for every bone: Translation X/Y/Z, Rotation (radians) X/Y/Z, Scale X/Y/Z.
```

Comment:
Sorry for being picky: could you add a new line between each header and the paragraph? When you do not do so, it will trigger a `MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines`. You receive that warning when you use VS Code as editor for markdown files.